contracts/validator-manager/PoAValidatorManager.sol

```diff
@@ -46,30 +45,34 @@ contract PoAValidatorManager is IPoAValidatorManager, ValidatorManager, OwnableU
     // solhint-disable-next-line no-empty-blocks
     function __PoAValidatorManager_init_unchained() internal onlyInitializing {}
 
-    // solhint-enable func-name-mixedcase
-
-    /**
-     * @notice See {IPoAValidatorManager-initializeValidatorRegistration}.
-     */
-    function initializeValidatorRegistration(
-        ValidatorRegistrationInput calldata registrationInput,
-        uint64 weight
-    ) external onlyOwner returns (bytes32 validationID) {
-        return _initializeValidatorRegistration(registrationInput, weight);
-    }
-
     // solhint-enable ordering
     /**
      * @notice See {IPoAValidatorManager-initializeEndValidation}.
      */
-    function initializeEndValidation(bytes32 validationID) external override onlyOwner {
-        _initializeEndValidation(validationID);
-    }
 
     /**
      * @notice See {IValidatorManager-completeEndValidation}.
      */
-    function completeEndValidation(uint32 messageIndex) external {
-        _completeEndValidation(messageIndex);
+    function handleInitializeValidatorRegistration(bytes32 validationID, uint64 weight, bytes calldata args) external {
+    }
+
+    function handleCompleteValidatorRegistration(bytes32 validationID) external {
+        // No-op
+    }
+
+    function handleInitializeEndValidation(bytes32 validationID, bytes calldata args) external {
+        // No-op
+    }
+
+    function handleCompleteEndValidation(bytes32 validationID) external {
+        // No-op
+    }
+
+    function handleInitializeValidatorWeightChange(bytes32 validationID, uint64 weight, bytes calldata args) external {
+        // No-op
+    }
+
+    function handleCompleteValidatorWeightChange(bytes32 validationID, bytes calldata args) external {
+        // No-op
```

Comment:
Wait... shouldn't POA still be able to cycle validators? Maybe I just don't have the full picture

Reply:
Yeah, these handlers should likely implement checks against the owner address/key. Keep in mind this PR is meant to demonstrate the architectural changes needed to implement ACP-99, so missing features are expected